evm/scripts/bench_evm384.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import re
+import subprocess
+import nanodurationpy as durationpy
+import csv
+import time
+import datetime
+import os
+import shutil
+import shlex
+
+# output paths should be mounted docker volumes
+RESULT_CSV_OUTPUT_PATH = "/evmraceresults"
+
+RESULT_CSV_FILENAME = "evmone_evm384.csv"
+
+EVMONE_BENCH_INFOS = [
+  {
+    "command": "/root/evmone-evm384-v1/build/bin/evmone-bench --benchmark_color=false --benchmark_min_time=5 /root/evm384_f6m_mul/build/v1-f6m_mul_bench.bin 00 74229fc665e6c3f4401905c1a454ea57c8931739d05a074fd60400f19684d680a9e1305c25f13613dcc6cdd6e6e57d0800000000000000000000000000000000",
+    "bench_name": "evm384-synth-loop-v1"
+  },
+  {
+    "command": "/root/evmone-evm384-v2/build/bin/evmone-bench --benchmark_color=false --benchmark_min_time=5 /root/evm384_f6m_mul/build/v2-f6m_mul_bench.bin 00 74229fc665e6c3f4401905c1a454ea57c8931739d05a074fd60400f19684d680a9e1305c25f13613dcc6cdd6e6e57d0800000000000000000000000000000000",
+    "bench_name": "evm384-synth-loop-v2"
+  }
+]
+
+
+"""
+root@472ab2fd1fc1:~/evm384_f6m_mul# /root/evmone-evm384-v2/build/bin/evmone-bench ~/evm384_f6m_mul/build/v2-f6m_mul_bench.bin "00" "74229fc665e6c3f4401905c1a454ea57c8931739d05a074fd60400f19684d680a9e1305c25f13613dcc6cdd6e6e57d0800000000000000000000000000000000"
+Benchmarking evmone
+
+2020-06-18 20:52:56
+Running /root/evmone-evm384-v2/build/bin/evmone-bench
+Run on (4 X 2294.68 MHz CPU s)
+CPU Caches:
+  L1 Data 32K (x2)
+  L1 Instruction 32K (x2)
+  L2 Unified 256K (x2)
+  L3 Unified 51200K (x2)
+-------------------------------------------------------------------------------------------------------
+Benchmark                                                Time           CPU Iterations UserCounters...
+-------------------------------------------------------------------------------------------------------
+/root/evm384_f6m_mul/build/v2-f6m_mul_bench.bin      18156 us      18156 us         39 gas_rate=322.266M/s gas_used=5.85118M
+"""
+
+
+def do_evmone_bench(evmone_bench_cmd):
+    evmone_cmd = shlex.split(evmone_bench_cmd)
+    print("running evmone benchmark...\n{}".format(evmone_bench_cmd))
+
+    stdoutlines = []
+    with subprocess.Popen(evmone_cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, bufsize=1, universal_newlines=True) as p:
+        for line in p.stdout: # b'\n'-separated lines
+            print(line, end='')
+            stdoutlines.append(line)  # pass bytes as is
+        p.wait()
+
+    timeregex = "\S+\s+(\d+) us"
```

Comment:
Can you take a note from @hugo-dc's #85 PR which uses the JSON output from evmone?

Reply:
this is working already, can refactor later.

Reply:
turned out it was parsing the gas number wrong, so did this now.